Enhance README with setup and usage instructions

darakhshannehal opened the pull request:
This PR addresses issue #2 by enhancing the README with the requested sections:

Prerequisites: Lists required software and R packages

Installation: Includes a single install.packages() command

How to Use This Repository: Brief workflow guidance with file references

License: Placeholder section for future specification, I will revise if a specific license is preferred.

Just a suggestion: would you be open to including a package loader like pacman or maybe positron (or maybe RStudio might be beginner-friendly enough and might suffice for now)? 

Let me know, I will update.

darakhshannehal committed:
Enhance README with setup and usage instructions

cecibaldoni commented on README.md:
```diff
@@ -1,13 +1,30 @@
+    -   `quarto`
+    -   `gapminder`
 
-You can find the slides [here](https://cecibaldoni.github.io/data-wrangling-tidyverse/#/title-slide)
```
I would leave this link, maybe at the beginning or at the end of the readme

cecibaldoni commented on README.md:
```diff
@@ -1,13 +1,30 @@
+
+To install the required packages, run the following command in your R console:
+
+\`\`\` r install.packages(c("tidyverse", "gapminder"))
```
What about

`install.packages(setdiff(c("tidyverse", "gapminder"), rownames(installed.packages())))`
This checks if the packages are already installed, and only install the missing ones
I am thinking of actually putting this at the very beginning of the script. What do you think?

darakhshannehal replied:
This seems fine.

darakhshannehal replied:
In README we can also add this, and it's a good practice in educational context.

cecibaldoni commented on README.md:
```diff
@@ -1,13 +1,30 @@
+
+## License
+
+License information to be added.
```
I would go for MIT!

cecibaldoni reviewed (changes requested):
Great job, thank you so much!
Do you want to change the few comments I left? Then I merge!

darakhshannehal committed:
Update README: install line, slide link, MIT license

darakhshannehal commented:
Thanks for the suggestions! 
I’ve updated the install line, added the slide link, and licensed the project under MIT.
Let me know if you'd like any further changes.

cecibaldoni commented:
It looks great! Thanks so much!
